shims/common/src/main/java/org/apache/hadoop/hive/thrift/HadoopThriftAuthBridge.java

```diff
@@ -557,11 +676,28 @@ public boolean process(final TProtocol inProt, final TProtocol outProt) throws T
         String mechanismName = saslServer.getMechanismName();
         userAuthMechanism.set(mechanismName);
         if (AuthMethod.PLAIN.getMechanismName().equalsIgnoreCase(mechanismName)) {
+/*
           remoteUser.set(endUser);
           return wrapped.process(inProt, outProt);
+*/
```

Comment:
Removing code, instead of commenting might improve readability of this next section whole section. Also a comment for why code can be commented, is not necessary anymore (despite this being an added feature) can help the reviewer ... me. :)